scalatags/src/main/scala/org/http4s/scalatags/instances/package.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package scalatags
+package instances
+
+package object instances {
+  object all extends AllInstances
+  object scalatags extends ScalatagsInstances
```

Comment:
Okay, yay.  This one is as simple as the scala-xml one I did... and as simple as I thought the non-circe ones would all be.  But maybe we should call it `FragInstance`.